wsd/Storage.cpp

```diff
@@ -697,7 +697,7 @@ std::unique_ptr<WopiStorage::WOPIFileInfo> WopiStorage::getWOPIFileInfoForUri(Po
             }
             else
             {
-                LOG_TRC("WOPI::CheckFileInfo redirected too many times\n");
+                LOG_WRN("WOPI::CheckFileInfo redirected too many times - URI [" << uriAnonym << "]\n");
```

Comment:
Each log is on its own line. No need to have `\n`.

Reply:
done